Dockerfile

```diff
@@ -8,6 +8,9 @@ ARG RESTIC_SHA256=e22208e946ede07f56ef60c1c89de817b453967663ce4867628dff77761bd4
 ARG GO_CRON_VERSION=0.0.2
 ARG GO_CRON_SHA256=ca2acebf00d61cede248b6ffa8dcb1ef5bb92e7921acff3f9d6f232f0b6cf67a
 
+RUN apt-get update -qq \
+ && apt-get install -yq unzip
```

Comment:
I'd rather have this together in one `RUN` instruction with the rclone extraction so we can install and uninstall unzip in the same layer.